app/models/court_case.rb

```diff
@@ -0,0 +1,35 @@
+class CourtCase
+  include ActiveModel::Serialization
+
+  attr_reader :id, :case_seq, :begin_date, :case_type, :case_info_number, :case_status, :agency_id
+
+  def build_from_nomis(court_case)
+
+    @case_info_number = court_case['caseInfoNumber']
+    @id = @case_info_number
+
+    @case_seq = court_case['caseSeq']
+    @begin_date = court_case['beginDate']
+    @case_type = court_case['caseType']
+    @case_status = court_case['caseStatus']
+
+    @agency_id = court_case['agency']['agencyId']
```

Comment:
Maybe use `dig` here just in case the `agency` key is missing in the JSON payload

Reply:
The agency is an important field here (it identify the Court), so we expect it to be there. I think it's better to raise a exception here so we can investigate it's missing.

Reply:
Okay 👍